backend/sessionManagementRoutes.js

```diff
@@ -1,4 +1,5 @@
 const passport = require('passport')
+const querystring = require('querystring')
```

Comment:
code in this and test shamelessly copied from prisonstaffhub that had been done previously